golang/pkg/builder/container/container_builder.go

```diff
@@ -219,9 +219,30 @@ func (dc *DockerContainerBuilder) WithTTY(tty bool) *DockerContainerBuilder {
 
 // Deletes the container with the given name if already exists.
 func (dc *DockerContainerBuilder) WithoutConflict() *DockerContainerBuilder {
-	if err := deleteContainer(dc.ctx, dc.containerName); err != nil {
+	containers, err := getContainer(dc.ctx, dc.containerName)
```

Comment:
Why the plural, it is just one container.

Reply:
true, but container would cause name-collision with an imported dependency

Reply:
fixed